app/react/Library/components/LibraryHeader.tsx

```diff
@@ -0,0 +1,138 @@
+import { Translate } from 'app/I18N';
```

Comment:
There's a small difference in style for desktop on the sort and counter for the library:
Current:
![image](https://user-images.githubusercontent.com/71732018/198687918-064a5ca3-6aa5-4a8e-8c58-3c9766760092.png)
PR:
![image](https://user-images.githubusercontent.com/71732018/198687957-c83b5dad-eb27-4ebc-9f68-d446c0a70991.png)

Reply:
I'm working on this and other feedback from @juanmnl about the disposition of the elements